ComfortZoneApp/app/src/main/java/com/example/comfortzone/Utils/ParseUtil.java

```diff
@@ -9,6 +9,7 @@
 import com.parse.ParseQuery;
 import com.parse.ParseUser;
 
+import java.util.ArrayList;
```

Comment:
Can we rename this class to something else like ComforLevelUtil? We are currently using Parse database but for a real life project, we can change the database later to be more scalable so its a good practice to name the Util based on what the Util does instead of naming it based on what kind of databse we are using.